lib/ExpressRedisCache/route.js

```diff
@@ -166,7 +166,11 @@ module.exports = (function () {
           /** if it's cached, display cache **/
 
           if ( cache.length ) {
-            res.contentType(cache[0].type || "text/html");
+            if (res.contentType) {
```

Comment:
I think we can avoid this conditional by using `res.header` which is supported by Express v4.x and Restify.